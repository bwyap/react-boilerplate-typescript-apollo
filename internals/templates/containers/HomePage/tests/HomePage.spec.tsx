/**
 * Test the HomePage
 */

import * as React from 'react';
import { render } from 'react-testing-library';
import 'react-testing-library/cleanup-after-each';
import { IntlProvider } from 'react-intl';

import { HomePage } from '../index';

describe('<HomePage />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <IntlProvider locale="en">
        <HomePage />
      </IntlProvider>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
