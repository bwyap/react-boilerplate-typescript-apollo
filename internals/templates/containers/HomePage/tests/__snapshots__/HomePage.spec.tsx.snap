// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> should render and match the snapshot 1`] = `
<article>
  <div
    style="text-align: center;"
  >
    <h2>
      <span>
        React Boilerplate - Now in Typescript with Apollo-Client!
      </span>
    </h2>
    <p>
      <span>
        A highly scalable, offline-first foundation with the best DX and a focus on performance and best practices
      </span>
    </p>
  </div>
</article>
`;
